### ISSUE TITLE
Create annotated bibliography

### ISSUE THREAD

**Issue by maneesha** [carpentries/instructor-training#1137]
There are a lot of studies cited in the Instructor Training curriculum.  it would be useful to have them all listed on one page, with a short description, for people who want to go back and reference them without having to remember where in the lesson the study was mentioned.

**Comment by ChristinaLK** [comment]
There's a list here but I'm not sure if it's kept up to date: https://carpentries.github.io/instructor-training/reference.html

**Comment by maneesha** [comment]
What's on that page looks like recommendations for additional reading.  I was thinking of an annotated bibliography of every source that is mentioned during the instructor training lessons.

**Comment by klbarnes20** [comment]
I definitely think an annotated bibliography for instructor training is a good idea. Perhaps this is something we can add when we do the curriculum update?

**Comment by Denubis** [comment]
In terms of bibliographies and jekyll, I have a remarkable amount of experience (onwork.edu.au, for example.) How were you thinking of populating it? Zotero group?

**Comment by ChristinaLK** [comment]
I think @karenword wants to do this as part of the next curriculum update!

**Comment by karenword** [comment]
Yes, and we are looking at Zotero using a group that @jt14den toyed with a while back. On the agenda for this week's meeting! @Denubis that bibliography is beautiful and interesting -- looks like a great model. If we need to structure our use of Zotero in a particular way to make this work well, it would be great to learn how to do that now.

**Comment by Denubis** [comment]
@karenword that bibliography was in paperpile, but isn't suitable for a huge team like this. The first call is: "Pull over during compile or pull dynamically on page load or pull on running a manual command?" If it's the manual command then I can pull some of my infrastructure from onwork to render pages and use citeproc and then all we have to do is make a zotero-md parser. ... But since this is annotated we also need to check that note-pulling-in works. 

@karenword Can you make a group, add me to it, and add no more than 5 different bibliographic sources? Preferably different in publication type (Journal, conference, book chapter, news, etc?) I'll try to noodle up a proof of concept in my free time for us to decide if this is worth doing properly. 

The other question is if you want any fancy faceted searching or just a big sorted list on a page, maybe with some headers by some sort of intrinsic criteria?

**Comment by Denubis** [comment]
I made an exceptionally abbreviated proof of concept: https://github.com/Denubis/pyzotero2lesson/tree/gh-pages/_bibliography demonstrating reading from zotero and writing to md. What would be the next step of a persuasive demo? Do you have a rough model for me to try to match? @brownsarahm suggested jekyll-scholar, so I also plan to investigate that in the context of github's new actions to see if there is an easy route for integration.

**Comment by karenword** [comment]
Thank you @Denubis! So to be clear, what this appears to do is create a repository that could be drawn upon to create a rendered page of the kind you demonstrate at onwork.edu.au. I suggest we hold off on further development for the moment -- we need to play around with our Zotero library to see how its features will best work for us, and try using it consistently. It is also worth considering the advantage of rendering a page here vs. linking people directly to Zotero, including prospects for maintenance. 
For the time being, I think the main thing we need to know is whether choices we make in using Zotero features/categories might limit the functionality of this script in any way. Otherwise, we will charge ahead with that library and reassess this when we have something that is more developed and ready to present.

**Comment by Denubis** [comment]
Let's start with the literals:
1. This demonstrates that the Zotero API works (yay)
2. This demonstrates that I can write bibilographic entries to Jekyll/Markdown so that we can do what we wish with them.

The primary thing that this allows us to do is manage cognitive load. Do we want to ease maintenance (one less tool) but require people to go to a new site? Do they know the note/abstract paradigms of zotero? What are our user profiles of people who will benefit from zotero? (These are genuine questions, BTW).

OR: Do we want to increase maintenance load, but point people at a website? Who does this benefit? Who would benefit from viewing notes and annotations on a website over zotero? Are there useful interactions with hypothes.is for your trainer training possible here, but not possible in zotero? 

Because this was literally the smallest demonstration of interaction that I could build quickly, I can't really answer your questions about limitations of zotero features. My intuition is that if we can build a model for managing goals, maintenance, and cognitive load we'll want me to build a second proof of concept to demonstrate functionality of our desired features. While onwork.edu.au just crossed 2000 citations, it is a grant-funded project with this as one of its fundamental mandates and the expected users are not particularly aware of group-based citation management systems. 

Agenda:
* To follow backwards design principles, can you folks state our desired goals and make some learner profiles? :) I think that will make the zotero/website/something else tension much clearer than anything else we can do.
* If we identify key zotero functionality for above, I will then need to demonstrate that I can interact with it so that decisions and tradeoffs can be informed.
